standalone-metastore/packaging/src/docker/Dockerfile

```diff
@@ -42,22 +42,46 @@ RUN echo ${BUILD_ENV}
 ARG HADOOP_VERSION
 ARG HIVE_VERSION
 
-RUN tar -xzvf /opt/hadoop-$HADOOP_VERSION.tar.gz -C /opt/ && \
-    rm -rf /opt/hadoop-$HADOOP_VERSION/share/doc/* && \
+RUN apt-get update && \
+    apt-get install -y wget
+
+RUN tar -xzv \
```

Comment:
@ayushtkn anything else we could cut?
````
du -h /opt/hadoop
481M
````
sh-5.1$ ls ./share/hadoop/common/
hadoop-common-3.4.1.jar  hadoop-kms-3.4.1.jar  hadoop-nfs-3.4.1.jar  hadoop-registry-3.4.1.jar	lib

418M	./share/hadoop/common/lib
423M	./share/hadoop/common